Marlin/src/gcode/geometry/G92.cpp

```diff
@@ -93,10 +93,37 @@ void GcodeSuite::G92() {
       COPY(coordinate_system[active_coordinate_system], position_shift);
   #endif
 
-  bool didA = parser.seenval('A');
-  if (didA) {
-    const float curr_z = current_position[Z_AXIS];
-    const float adjust_z = parser.value_axis_units(Z_AXIS);
+  bool doZAdjust = false;
```

Comment:
A flag to show Zadjustment was done, either "G92 Ann" or "G92 Bnn". Needed for sync_plan_position later.